Sources/Segment/Settings.swift

```diff
@@ -37,6 +39,8 @@ public struct Settings: Codable {
         self.plan = try? values.decode(JSON.self, forKey: CodingKeys.plan)
         self.edgeFunction = try? values.decode(JSON.self, forKey: CodingKeys.edgeFunction)
         self.middlewareSettings = try? values.decode(JSON.self, forKey: CodingKeys.middlewareSettings)
+        self.metrics = try? values.decode(JSON.self, forKey: CodingKeys.metrics)
+        self.consentSettings = try? values.decode(JSON.self, forKey: CodingKeys.consentSettings)
```

Comment:
Where is this `CodingKeys.consentSettings` defined?

Reply:
down below as 

```
    enum CodingKeys: String, CodingKey {
        case integrations
        case plan
        case edgeFunction
        case middlewareSettings
        case metrics
        case consentSettings
    }
 ```